source/configure/calls-deployment.rst

```diff
@@ -312,6 +312,23 @@ Some caveats apply here. Web socket events (for example: emoji reactions, hand r
 
 In general, ``rtcd`` is the preferred solution for a performant and scalable deployment. With ``rtcd``, the Mattermost server will be minimally impacted when hosting a high number of calls.
 
+Horizontal scalability
+~~~~~~~~~~~~~~~~~~~~~~
+
+The supported way to enable horizontal scalability for Calls is through a form of DNS based load balancing. This can be achieved regardless of how the ``rtcd`` service is deployed (barebone instance, Kubernetes or else).
```

Comment:
```suggestion
The supported way to enable horizontal scalability for Calls is through a form of DNS based load balancing. This can be achieved regardless of how the ``rtcd`` service is deployed (bare bone instance, Kubernetes, or an alternate way).
```